actions/inventory.go

```diff
@@ -39,7 +40,14 @@ type Collectors struct {
 func InitCollectors(trace bool) *Collectors {
 	return &Collectors{
 		Inventory: utils.NewLshwCmd(trace),
-		Drives:    []DriveCollector{utils.NewSmartctlCmd(trace)},
+		Drives: []DriveCollector{
+			utils.NewSmartctlCmd(trace),
+			utils.NewLsblkCmd(trace),
+		},
+		DriveCapabilities: []DriveCapabilityCollector{
```

Comment:
Probably a bit late in the game for this type of question...     Why did we opt to make DriveCapabilities a 'top level' collector rather than just an additional collector (or collector(s)) added to Drives based upon the type of Drives found during the initial Inventory?

Reply:
They are separate since not all collectors can return drives and drive capabilities,
and and so extending the existing `DriveCollectors` interface with a `DriveCapabilities()` method wouldn't be nice to maintain - it forces all DriveCollector utils to implement the `DriveCapabilities()` method even if they don't support it. 

A smaller scoped interface seems ideal for this case. Hope that answers the question?

Reply:
Yeah I think that clarifies things for me.